cmd/notary/tuf.go

```diff
@@ -142,7 +142,10 @@ func (t *tufCommander) AddToCommand(cmd *cobra.Command) {
 	cmdReset.Flags().BoolVar(&t.resetAll, "all", false, "Reset all changes shown in the status list")
 	cmd.AddCommand(cmdReset)
 
-	cmd.AddCommand(cmdTUFPublishTemplate.ToCommand(t.tufPublish))
+	cmdTUFPublish := cmdTUFPublishTemplate.ToCommand(t.tufPublish)
+	cmdTUFPublish.Flags().StringVar(&t.rootKey, "rootkey", "", "Root key to initialize the repository with")
```

Comment:
hm it depends on whether we want lazy initialization to support this (we'd have to inject the root key into [this call to `Initialize`](https://github.com/docker/notary/pull/1105/files/6a1f1c1f5a528906d486d3d0a21afef98658264f#diff-1cd41a2611da49ce63630d5a51bd82c5R847)), and it will get messy with the auto-publish flag.

I'm ok with removing this for now, since `notary init` will still allow for this functionality.  We can figure out the best way to handle this for lazy init on a followup PR 👍